src/video/SDL_blit_A.c

```diff
@@ -23,6 +23,11 @@
 #include "SDL_video.h"
 #include "SDL_blit.h"
 
+// Workaround for lack of MMX intrinsic support for non-SSE2 targets
```

Comment:
nit: Rest of file appears to use C-style comments (`/* Comment */`), even for single-line comments